### DISCUSSION TITLE
Support for other languages

### DISCUSSION THREAD

**Discussion by llanni** [mozilla/firefox-translations#67] [Q&A] [answered]
Hi,

First of all congratulations for the great idea.
Are you planning to add also other languages for the offline translations, specifically what about Dutch (NL)?

Thank you

**Comment by kpu** [comment, marked as answer]
French and Polish are actively being worked on with ETA in a month.  There is a project proposal under consideration by the EU that would scale the project to many more languages, including Dutch, but I won't know if the project is funded (much less starting) for at least 2 months.

**Comment by evgenyrp** [comment]
Dutch is the next candidate for training because it's a fairly high-resource and still should be supported by the training pipeline, unlike Asian languages. However, we're focusing our effort on retraining Russian, Portuguese and Italian at the moment to get higher quality and test all the latest improvements in the pipeline.

**Comment by skarmanami** [comment]
Hi! Are you going to add Swedish and Finnish?

**Comment by snovotill** [comment]
Hi Folks, Chinese is actually the most important language to implement because China has the second largest economy in the world and they don't speak English, so there is tremendous need to communicate back and forth. I see this is not being worked on and am thinking this is not an oversight; Is there some fundamental limitation that prevents this?
...Stepan

**Comment by kpu** [comment]
The High Performance Language Technologies project has been funded by the EU.  Brexit has made everything worse, but the UK should be funding our participation provided the paperwork can be sorted.  The EUR 5 million project starts 1 September 2022 and lasts for 3 years.  It will take some time to produce results and get a larger scale pipeline running.  It includes support for far more languages (and not just EU ones), reproducible training, enhancing OPUS, language models for non-English languages, and mining 7 petabytes of web data.
